integration/container/run_linux_test.go

```diff
@@ -289,3 +289,40 @@ func TestMacAddressIsAppliedToMainNetworkWithShortID(t *testing.T) {
 	c := container.Inspect(ctx, t, apiClient, cid)
 	assert.Equal(t, c.NetworkSettings.Networks["testnet"].MacAddress, "02:42:08:26:a9:55")
 }
+
+func TestStaticIPOutsideSubpool(t *testing.T) {
```

Comment:
Perhaps it's useful to add a description to the test (based on your PR description on GitHub); this seems like one of those cases where context may be useful to know "why" this test is.